src/Sylius/Bundle/AdminBundle/Event/AdministratorStatusMenuBuilderEvent.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\AdminBundle\Event;
+
+use Knp\Menu\FactoryInterface;
+use Knp\Menu\ItemInterface;
+use Sylius\Bundle\UiBundle\Menu\Event\MenuBuilderEvent;
+
+class AdministratorStatusMenuBuilderEvent extends MenuBuilderEvent
```

Comment:
```suggestion
final class AdministratorStatusMenuBuilderEvent extends MenuBuilderEvent
```

Reply:
Why do we need this class?

Reply:
This class adds a Menu Builder Event to the Administrator status page so someone could add buttons like `Clear cache`